### PR TITLE
docs: add inference exemple in readme

### PR DESCRIPTION
**Pull request by MateoLostanlen**
The readme is missing a clear and simple example to predict if there is fire on a given image

I also move the example of how to use our dataset which was misplaced in the readme

### CODE REVIEW

**Review by chloeskt — COMMENTED** [review]
Hi @MateoLostanlen, thank you for this PR ! It is very useful to have an short example of how to use the lib to make the inference 🤗

File: README.md [hunk 1]
```diff
@@ -70,8 +70,37 @@ conda install -c pyronear pyrovision
...
+imT = tf(im)
+
+with torch.no_grad():
+    pred = model(imT.unsqueeze(0))
+    pred = torch.sigmoid(pred).item()
+    
+if pred < 0.5:
+    print("No fire detected. Fire probability: {:.2%}".format(pred))
+else:
+    print("Fire detected! Fire probability: {:.2%}".format(pred))
```
Comment (frgfm) [inline README.md]: I suggest removing this to make the snippet shorter

File: README.md [hunk 2]
```diff
@@ -70,8 +70,37 @@ conda install -c pyronear pyrovision
...
+model = rexnet1_0x(pretrained=True).eval()
+
+# Predict
+im_file = 'im.jpg'
+im = Image.open(im_file).convert('RGB')
+imT = tf(im)
+
+with torch.no_grad():
+    pred = model(imT.unsqueeze(0))
+    pred = torch.sigmoid(pred).item()
```
Comment (frgfm) [inline README.md]: Suggestion to make this shorter by replacing this line by:
```
is_wildfire = torch.sigmoid(pred).item() >= 0.5
```

File: README.md [hunk 3]
```diff
@@ -70,8 +70,37 @@ conda install -c pyronear pyrovision
...
+tf = transforms.Compose([transforms.Resize(size=(img_size)),
+                              transforms.CenterCrop(size=img_size),
+                              transforms.ToTensor(),
+                              normalize
+                              ])
+
+model = rexnet1_0x(pretrained=True).eval()
+
+# Predict
+im_file = 'im.jpg'
```
Comment (frgfm) [inline README.md]: you can remove this

File: README.md [hunk 4]
```diff
@@ -70,8 +70,37 @@ conda install -c pyronear pyrovision
...
+im = Image.open(im_file).convert('RGB')
```
Comment (frgfm) [inline README.md]:
```
im = Image.open("path/to/your/image.jpg").convert('RGB')
```
to make it more obvious :)

**Review by frgfm — COMMENTED** [review]
Thanks for the PR! A few edit suggestion to make the snippet more concise but it's nitpicking :)

**Comment by MateoLostanlen** [comment]
@frgfm thanks for your review, I'v made the changes

File: README.md [hunk 5]
```diff
@@ -70,8 +70,26 @@ conda install -c pyronear pyrovision
...
+normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
+
+tf = transforms.Compose([transforms.Resize(size=(448)), transforms.CenterCrop(size=448),
+                         transforms.ToTensor(), normalize])
+
+model = rexnet1_0x(pretrained=True).eval()
+
+# Predict
+im = Image.open("path/to/your/image.jpg").convert('RGB')
+imT = tf(im)
```
Comment (frgfm) [inline README.md]: we can move this in the context while we're at it :ok_hand:

File: README.md [hunk 6]
```diff
@@ -70,8 +70,26 @@ conda install -c pyronear pyrovision
...
+
+with torch.no_grad():
+    is_wildfire = torch.sigmoid(pred).item() >= 0.5
```
Comment (frgfm) [inline README.md:92]: I think you're missing `pred = model(tf(im).unsqueeze(0))` over here :)

**Review by frgfm — CHANGES_REQUESTED** [review]
Small typo to fix :sweat_smile:

File: README.md (hunk 5 above)
Comment (MateoLostanlen) [reply README.md:89]: remove* ?

File: README.md (hunk 6 above)
Comment (MateoLostanlen) [reply README.md:92]: oups :)

File: README.md (hunk 5 above)
Comment (MateoLostanlen) [reply README.md]: add the tf on the same line ?

**Review by frgfm — APPROVED** [review]
Thanks for the edits!

**Pull request merged**